core/tests/protractor/explorationFeedback.js

```diff
@@ -49,10 +49,12 @@ describe('ExplorationFeedback', function() {
   });
 
   beforeEach(function() {
-    users.createUser('user1@ExplorationFeedback.com',
-                     'creatorExplorationFeedback');
-    users.createUser('user2@ExplorationFeedback.com',
-                     'learnerExplorationFeedback');
+    users.createUser(
+      'user1@ExplorationFeedback.com',
+      'creatorExplorationFeedback');
```

Comment:
> I don't think any rule pertaining to this was changed. Shouldn't we have an indent-by-two in this line?

FYI I think the issue was just that the email address was at the end of line 52. But now you've added a line break after '(' so it should be fine.

Reply:
Okay, got it! Thanks!